### DISCUSSION TITLE
Cors issue on the components tab - Please help

### DISCUSSION THREAD

**Discussion by verlindb** [DependencyTrack/dependency-track#985] [General]
Hi all,

We noticed some cors issue on the components functionality (container version api & frontend latest version )
even  with the default configuration setting from the documentation
which has default cors disabled is guess

![issue dependencytrack](https://user-images.githubusercontent.com/29273350/112443665-83e8c380-8d4d-11eb-8a62-db71a19944bf.PNG)

**Comment by stevespringett** [comment]
CORS is configurable. See https://docs.dependencytrack.org/getting-started/configuration/
